src/doomstat.h

```diff
@@ -245,6 +245,11 @@ typedef struct
 	UINT8 numGradedMares;   ///< Internal. For grade support.
 	nightsgrades_t *grades; ///< NiGHTS grades. Allocated dynamically for space reasons. Be careful.
 
+	// miru: to use custom mapheaderinfo options, we need to add them to the struct first
+	UINT8 levelwipe;
+	UINT8 postlevelwipe;
+	UINT8 wipecolor;
```

Comment:
Frankly it's, without padding at least, going to add merely 3 bytes extra per header in memory. So 1035*3 = 3105 bytes or ~3kb extra in memory, right? Add another kb or so if we include padding, it's not THAT much?